### ISSUE TITLE
Action - Ensure color is not the sole means of communicating information - (2036569746)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5554]
### Module: 
 04 Action 

### Violation: 
 Ensure color is not the sole means of communicating information 

### WCAG Reference: 
- [1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) 

### Instance ID: 
 2036569746 

### Severity: 
 10 

### Description: 
 [Issue]
The "blue" indicator uses color alone to visually indicate that the notification is present.

[User Impact]
Users with color blindness will have difficulty identifying the colors used to convey this information.

[Code Reference] 
 ```
<button aria-busy="false" aria-disabled="false" aria-label="Test 10 action with indicator, for display purposes only" class="button button--text-visible">
(...)
<div class="text-container text-container--visible">Notification</div>
</button>
 ``` 

### Note: 
 [Suggestion]
Ensure color is not the sole means of communicating information. Color can be used as long as other visual indicators of the information are also present. Common additional visual indicators include underlining, bolding, additional on-screen text conveying the information, or change in shape or size.

[Compliant Code Example] 
 ```
N/A
 ```
 

### Media Type: 
 Color and Contrast 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
This issue may have been a testing-related issue.

In the [test #10](https://geospatialem.github.io/calcite-samples/action) where the indicator is present, the tester might not have  context of the non-indicator option, which does not have a blue-colored dot present.
